src/app/contextBroker/contextBroker.cpp

```diff
@@ -1771,14 +1763,14 @@ int main(int argC, char* argV[])
     LM_T(LmtHttps, ("httpsKeyFile:  '%s'", httpsKeyFile));
     LM_T(LmtHttps, ("httpsCertFile: '%s'", httpsCertFile));
 
-    restInit(rsP, ipVersion, bindAddress, port, mtenant, connectionMemory, maxConnections, reqPoolSize, rushHost, rushPort, allowedOrigin, httpsPrivateServerKey, httpsCertificate);
```

Comment:
Removing connectionMemory, maxConnections and reqPoolSize from restInit() we are lossing the capability of set these parameteres based on CLI values, aren't we?

In addition, I don't understand why rushPort has been also removed...